builds/misc/images.yaml

```diff
@@ -26,7 +26,9 @@ jobs:
         name: build
         displayName: Build ($(Build.Configuration)) dotnet artifacts
 
-      - template: templates/build-broker-watchdog.yaml
```

Comment:
This used to be one template but I split it up so that we can re-use it. We shouldn't reuse it though for perf tests until we start testing arm64.